Controllers/IndexController.cs

```diff
@@ -176,6 +173,22 @@ public IActionResult Profile()
                 }
             }
 
+            if (members.Count == 0)
+            {
+                var emptyMem = new MemberUpdateViewModel
+                {
+                    MemberId = null,
+                    Name = "",
+                    Notes = "",
+                    Restrictions = ConsiderationsLists.RestrictionsList,
+                    Goals = ConsiderationsLists.GoalsList,
+                    Cuisines = ConsiderationsLists.CuisinesList,
+                    ShouldDelete = false
+                };
+
+                viewModelMembers.Add(emptyMem);
+            }
+
```

Comment:
If we somehow have a family that has a family size of 0 this should mitigate that problem